FBSDKMessengerShareKit/FBSDKMessengerShareKit.xcodeproj/project.pbxproj

```diff
@@ -534,41 +534,49 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9CAE101E1B15355200163BDA /* FacebookSDK-LogicTests.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.sdk.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
we use xcconfig files, can you move this to FBSDKMessengerShareKit/Configurations/FacebookSDK-Project.xcconfig